### DISCUSSION TITLE
How do we automate publishing videos?

### DISCUSSION THREAD

**Discussion by laurenwigmore** [life-itself/community#974] [General]
When I have a rough and ready video that I want to share with the community, or a subset of the community, how is best to automate publishing the videos and where is best to share (currently we share to YouTube) in your opinion?

**Comment by martinprogrammer** [comment]
@laurenwigmore  If I understand correctly there are 3 questions here:

1) Editing
In my opinion, it's a manual process - I use free software - DaVinci Resolve.  (It could be Final Cut Pro, Adobe Premiere, Sony Vegas Pro)
If you need a workflow - I'll add it.

2) Publishing on a video sharing platform
It's Youtube, Vimeo, Twitch ...

3) Sharing the video
Either as a link, or maybe embedded within an iFrame.
It works in Obsidian too and can be published -  <iframe src="https://www.youtube.com/embed/NPYArp-hhsM"></iframe>

**Comment by james-baker** [comment]
Oof, same here. I tried to use Zapier to automate uploading files to YouTube, and failed miserably as there's some "has to upload in <5 seconds" timeout even for paid Zapier accounts.
The manual flow for publishing on Youtube isn't TOO obnoxious, so that's what I currently do too.

**Reply by rufuspollock** [reply]
That's useful to hear as we'd been thinking of trying zapier.

Lauren (and we) were asking here because our current process seems pretty manual and cumbersome especially for "rough" videos where we want to get something up quickly ... 

Our current process seems a bit heavy duty - see https://lifeitself.org/tao/communications/producing-a-podcast-episode-guide
